petastorm/reader_impl/pytorch_shuffling_buffer.py

```diff
@@ -90,37 +89,36 @@ class BatchedNoopShufflingBuffer(BatchedShufflingBufferBase):
 
     def __init__(self, batch_size=1):
         super(BatchedNoopShufflingBuffer, self).__init__(batch_size=batch_size)
-        self._batches = []
-        self._num_samples = 0
-        self.store = deque()
         self._size = 0
-
-    def _make_batch(self):
-        # TODO: Add test for the zip
-        batch = [torch.cat(b, 0) for b in zip(*self._batches)]
-        if self._num_samples > self.batch_size:
-            leftover = [b[self.batch_size:] for b in batch]
-            batch = [b[:self.batch_size] for b in batch]
-            self._batches = [leftover]
-        else:
-            self._batches = []
-        self._num_samples -= min(self._num_samples, self.batch_size)
-        self.store.append(batch)
+        self._buffer = []
+        self._done_adding = False
+        self._batch_i = 0
```

Comment:
Perhaps a more self explanatory name instead of _batch_i? Something like `first_unconsumed_row` or something along the lines?

Reply:
updated with name and comments.